geo/src/algorithm/sweep/proc.rs

```diff
@@ -297,43 +297,6 @@ impl<C: Cross + Clone> Sweep<C> {
         true
     }
 
-    #[inline]
-    pub(super) fn with_prev_active<F: FnOnce(&Segment<C>) -> R, R>(
-        &self,
-        c: &Crossing<C>,
-        f: F,
-    ) -> Option<R> {
-        debug_assert!(c.at_left);
-        // {
-        //     // Expensive debug block
-        //     debug!("with_prev_active: {c:?}");
-        //     debug!("previous:");
-        //     self.active_segments.previous_find(&c.segment, |aseg| {
-        //         debug!("\t{geom:?}", geom = aseg.0.geom());
-        //         false
-        //     });
-        //     debug!("next:");
-        //     self.active_segments.next_find(&c.segment, |aseg| {
-        //         debug!("\t{geom:?}", geom = aseg.0.geom());
-        //         false
-        //     });
-        // }
```

Comment:
Instead of commenting it out, what if we put this logic behind a debug assertions flag? https://stackoverflow.com/a/39205417

Reply:
This commented out code was altogether deleted, so I don't think there's anything further for us to do here.

Reply:
Wow I completely misread the diff